### ISSUE TITLE
Training Model Issue

### ISSUE THREAD

**Issue by swandawg17** [EthoML/VAME#201]
Hi there, I am new to using vame and I am having issues training the model, I keep getting the following error whether I try and use the pipeline or by following the vame step-by-step guide. I was able to train using the sample data, but when I try and use my own data (from DeepLabCut) this occurs. I would appreciate any help!

2026-01-12 12:26:48.814 INFO  --- [MainThread] vame.model.rnn_vae : 589 : TensorBoard logging enabled. Log directory: C:\Users\HP_Te\trimmed_pipeline_example\logs\tensorboard\VAME
2026-01-12 12:26:48.814 INFO  --- [MainThread] vame.model.rnn_vae : 590 : To view logs, run: tensorboard --logdir=C:\Users\HP_Te\trimmed_pipeline_example\logs\tensorboard --port=6006
2026-01-12 12:26:48.815 INFO  --- [MainThread] vame.model.rnn_vae : 595 : Using CUDA
2026-01-12 12:26:48.815 INFO  --- [MainThread] vame.model.rnn_vae : 596 : GPU active: True
2026-01-12 12:26:48.815 INFO  --- [MainThread] vame.model.rnn_vae : 597 : GPU used: NVIDIA GeForce RTX 5070 Laptop GPU
2026-01-12 12:26:48.815 INFO  --- [MainThread] vame.model.rnn_vae : 642 : Latent Dimensions: 30, Time window: 30, Batch Size: 256, Beta: 1, lr: 0.0005

2026-01-12 12:26:49.198 INFO  --- [MainThread] vame.model.rnn_vae : 700 : Model graph logged to TensorBoard
2026-01-12 12:26:49.214 INFO  --- [MainThread] vame.model.dataloader : 52 : Compute mean and std for temporal dataset.
2026-01-12 12:26:49.215 INFO  --- [MainThread] vame.model.dataloader : 62 : Initialize train data. Datapoints 16191
2026-01-12 12:26:49.230 INFO  --- [MainThread] vame.model.dataloader : 64 : Initialize test data. Datapoints 1798
2026-01-12 12:26:50.403 INFO  --- [MainThread] vame.model.rnn_vae : 765 : Scheduler step size: 100, Scheduler gamma: 0.20

2026-01-12 12:26:50.403 INFO  --- [MainThread] vame.model.rnn_vae : 781 : Start training...
2026-01-12 12:26:50.404 INFO  --- [MainThread] vame.model.rnn_vae : 83 : Training Model:   0%|          | 0/99 [00:00<?, ?epoch/s]
2026-01-12 12:27:09.521 | WARNING  | vame.model.rnn_vae:cluster_loss:118 - C:\Users\HP_Te\anaconda3\envs\vame\Lib\site-packages\vame\model\rnn_vae.py:118: UserWarning: torch.linalg.svd: During SVD computation with the selected cusolver driver, batches 0 failed to converge. A more accurate method will be used to compute the SVD as a fallback. Check doc at https://pytorch.org/docs/stable/generated/torch.linalg.svd.html (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\pytorch\aten\src\ATen\native\cuda\linalg\BatchLinearAlgebraLib.cpp:708.)
  U, sv_2, Vh = torch.linalg.svd(gram_matrix, full_matrices=False)

2026-01-12 12:27:09.522 INFO  --- [MainThread] vame.model.rnn_vae : 83 : Training Model:   0%|          | 0/99 [00:19<?, ?epoch/s]
2026-01-12 12:27:09.523 ERROR --- [MainThread] vame.model.rnn_vae : 983 : An error occurred: linalg.svd: The algorithm failed to converge because the input matrix is ill-conditioned or has too many repeated singular values (error code: 256).

**Comment by luiztauffer** [comment]
hey @swandawg17 , I'm happy you seem to have solved your issue. But would you mind describing what was the solution to your specific problem here? It might be helpful to other people in the future. Thanks!

**Comment by swandawg17** [comment]
I realized I was not using the filtered data from DeepLabCut, when I switched to the filtered data csv it fixed the issue!
